python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/vxlan-interface.j2

```diff
@@ -65,6 +65,11 @@ interface Vxlan1
    vxlan vlan {{ vlan.id }} flood vtep {{ vlan.flood_vteps | join(' ') }}
 {%         endif %}
 {%     endfor %}
+{%     for vlan in vxlan_config.vxlan.vlans | arista.avd.natural_sort('id') %}
+{%         if vlan.flood_group is arista.avd.defined %}
```

Comment:
Please check on EOS if we can have flood vteps on a vlan together with a flood group. If not, we could combine this into the for loop above, to avoid looping twice.